package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "marktext",
-  "version": "0.9.25",
+  "version": "0.10.15",
```

Comment:
Do you want to prepublish this version? if not, there is need to change this version number.